src/GregClient/NuGet/tools/Install.ps1

```diff
@@ -0,0 +1,9 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
is this script actually called somewhere here? or is `install`l called on restore automatically somehow?